index.html

```diff
@@ -17,202 +18,174 @@
+                <a class="nav-link" href="#featured-speakers">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">CC Campaign</a>
+              </li>
+            </ul>
+            <form class="d-flex mt-3" role="search">
+              <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+              <button class="btn btn-outline-success" type="submit">Search</button>
+            </form>
+          </div>
         </div>
       </div>
+    </nav>
+
+  </div>
+
+  <nav class="nav-desktop2 ">
+
+    <div class="nav-desktop2-margin">
+      <a href="https://www.facebook.com"><i class="bi bi-facebook"></i></a>
+      <a href="https://twitter.com/?lang=es"><i class="bi bi-twitter"></i></a>
+      <a href="about.html">Logout</a>
+      <a href="#about">Mypage</a>
+      <a href="#contact">English</a>
     </div>
+
+    
+
+
+
+  </nav>
+  <nav class="nav-desktop ">
+    <h5 class="offcanvas-title" id="offcanvasNavbarLabel"><img src="./images/Logo.png" alt=""></h5>
+    <a href="about.html">About </a>
+    <a href="#about">Program</a>
+    <a href="#contact">Join</a>
+    <a href="#portfolio">Sponsor</a>
+    <a href="#portfolio">News</a>
+    <a href="#portfolio">CC Campaign</a>
+
+
   </nav>
   <section class="background-intro">
-    <div class="container-md cocogoose">
-      <h2 class="h2-intro">"Hello!! Sharing world of Knowledge!!"</h2>
-      <h1 class="h1-intro">Web Programming course's for everyone 2023</h1>
-    </div>
-    <div class="container containerpro-1">
-      <p>
-        A joyful program that believe in the peer to peer programming to learn, creating a positive work with people all
-        around the world.
-      </p>
+    <div class="container cocogoose">
+
+      <div class="margin-text1">
+        <h2 class="h2-intro">"Hello!! Sharing world of Knowledge!!"</h2>
+        <h1 class="h1-intro">Web Programming course's for everyone 2023</h1>
+      </div>
+      
     </div>
-    <div class="container-md">
-      <p class="p-intro">2023.03.17(MON) ~ 18(TUE)</p>
-      <p class="p-intro2"> @National Museum of Bogotá</p>
+<div class="margin-text2">
+  <div class="containe containerpro-1">
+    <p>
+      A joyful program that believe in the peer to peer programming to learn, creating a positive work with people all
+      around the world.
+    </p>
+  </div>
+</div>
+    
+    <div class="margin-text2">
+      <div class="container">
+        <p class="p-intro">2023.03.17(MON) ~ 18(TUE)</p>
+        <p class="p-intro2"> Neiva-Huila-Colombia</p>
+      </div>
+
```

Comment:
![image](https://user-images.githubusercontent.com/111823606/233341034-b42cb24d-be2b-466d-9805-e4cd9e513619.png)
Hi, for the "2023.03.17....." I applied the cocogoose font, and for the "@ Neiva-......" I applied the Lato font, I hope that this will work, thanks!!!